src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_AutoSuggestBox.cs

```diff
@@ -56,6 +56,22 @@ public async Task When_Text_Changed_And_Focused_Should_Open_Suggestion_List()
 			SUT.IsSuggestionListOpen.Should().BeTrue();
 		}
 #endif
+		[TestMethod]
+		public async Task When_Text_IsNull()
+		{
+			var sut = new AutoSuggestBox();
+
+			WindowHelper.WindowContent = sut;
+
+			await WindowHelper.WaitForIdle();
+
+			sut.Text = null;
```

Comment:
This isn't supposed to work. It should be crashing in UWP.

Reply:
I'm pretty sure it does.

Reply:
Here is UWP behavior I'm seeing.

![image](https://user-images.githubusercontent.com/31348972/161823617-ad4aae0b-4346-4ca9-afb3-9e1debb727b9.png)